src/main/resources/application.base.conf

```diff
@@ -13,12 +13,13 @@ source = {
 }
 
 consignmentapi = {
+  useIamAuth = true
   profile = "slick.jdbc.PostgresProfile$"
   db {
     connectionPool = "HikariCP"
     driver = "org.postgresql.Driver",
     url = "jdbc:postgresql://"${DB_ADDR}":"${DB_PORT}"/consignmentapi?ssl=true&sslrootcert=/api/rds-ca-2019-root.pem&sslmode=verify-full",
     user = ${DB_USER}
-    password = ${DB_PASSWORD}
+    registerMbeans = true
```

Comment:
What does this line do?

Reply:
We're setting the password using `hikariDataSource.ds.getHikariConfigMXBean.setPassword` This config option is needed to allow you to do it. https://github.com/brettwooldridge/HikariCP/wiki/MBean-(JMX)-Monitoring-and-Management

Reply:
Probably worth a comment, since it's not very obvious.

Reply:
I've added one in.